natter-api/src/main/java/com/manning/apisecurityinaction/controller/ModeratorController.java

```diff
@@ -14,9 +14,11 @@ public ModeratorController(Database database) {
   }
 
```

Comment:
Moderator code review:

1. Authorization check is missing
2. spaceId should not be added in url and it can be added if it is encrypted
3. msgId should not be added in url and it can be added if it is encrypted
4. There is no check that the spaceId is owned by logged in user, so hacker can delete post from any space
5. There is no check that the msgId is owned by logged in user, so hacker can delete post from any space
6. Description of function and parameters is not added